src/internal/deploy/cmds/cmds.go

```diff
@@ -831,21 +850,33 @@ If <object store backend> is \"s3\", then the arguments are:
 	commands = append(commands, cmdutil.CreateAlias(deployAmazon, "deploy aws"))
 
 	deployMicrosoft := &cobra.Command{
-		Use:   "{{alias}} <container> <account-name> <account-key> <disk-size>",
+		Use:   "{{alias}} <disk-size> <container> <account-name> <account-key>",
```

Comment:
Nit: should `container`, `account-name` and `account-key` be in square braces (just based on the aws/google help text)?